programmers/제일-작은-수-제거하기.js

```diff
@@ -0,0 +1,6 @@
+function solution(arr) {
+    
+    arr.length > 1 ? arr.splice(arr.indexOf(Math.min(...arr)), 1) : arr = [-1];
```

Comment:
```suggestion
    
    arr.length > 1 ? arr.splice(arr.indexOf(Math.min.apply(null, arr)), 1) : arr = [-1];
```
정말 잘 푸셨네요bb
배열을 풀어서 최솟값을 알 수도 있지만
위처럼 apply를 사용하면 배열안에서도 찾을 수 있습니다 알아두시면 좋을 것 같아요ㅎㅎ

Reply:
하. . apply 사용해보려고했는데 파라미터가 너무 헷갈렷어요. 다시 공부해보겠습니다!